Wiki/NewToFvwm/WhyFvwm/index.md

```diff
@@ -45,15 +53,19 @@ by lots of core users. What is it about fvwm that users like?
   while using modern software, and fvwm-crystal, which gives a
   nice sleek look.
 
-Fvwm is quite versatile, but in order to get the most out of
+**Fvwm is versatile**, but in order to get the most out of
 fvwm, you need to be willing to take the time to learn its
 command syntax (which is just the configuration file syntax).
 As you learn how to tell fvwm how to handle your windows,
 you can tweak fvwm into your own dream window manager. Okay,
 there are limits as to what fvwm can do, but within its
 structure, there is a lot of power.
 
-# What are fvwm's weaknesses?
+<i><b>I like fvwm’s approach</b> - no defaults and everything is
+up to the user. I can hardly imagine myself using anything else.
+Even with my lame coding style, fvwm meets my needs perfectly.</i>
+
```

Comment:
Just like with never use `<b>`, never use `<i>`, as this doesn't have meaning outside of visual, instead you want an _emphasis_, so use `<em>...</em>` instead of `<i>...</i>`.

Reply:
Great, the WhyFvwm link is there.
Thanks, now I know which HTML code to use. There are so many choices.